libs/deserializer/src/test/java/de/cyface/deserializer/BinaryFormatDeserializerTest.java

```diff
@@ -184,6 +204,188 @@ void test() throws IOException, InvalidLifecycleEvents, UnsupportedFileVersion {
         }
     }
 
+    /**
+     * This test generates random sensor values within the supported value range and deserializes the bytes to ensure
+     * the values did not change.
+     */
+    @DisplayName("Happy Path test for the serialization and deserialization of 3d points.")
+    @Test
+    void testSerializeDeserialize() throws IOException, InvalidLifecycleEvents {
+
+        // Arrange - Events: start, stop (1 track)
+        final var batches = 100;
```

Comment:
This test can also be used as a sample implementation for others who want to use our serialization.